docs/site/BelongsTo-relation.md

```diff
@@ -43,6 +43,28 @@ related routes, you need to perform the following steps:
 This section describes how to define a `belongsTo` relation at the model level
 using the `@belongsTo` decorator to define the constraining foreign key.
 
+The definition of the `belongsTo` relation is inferred by using the `@belongsTo`
+decorator. The decorator takes in a function resolving the target model class
+constructor and designates the relation type. It also calls `property()` to
+ensure that the decorated property is correctly defined. Usage of
+belongsTo relation is similar to legacy [datasource juggler](https://github.com/strongloop/loopback-datasource-juggler).
+
+The `@belongsTo` decorator takes three parameters:
+
+- `target model class` (mandatory)
+- `relation definition` (optional) - has three attributes, keyFrom,
+  keyTo, name
+  - `keyFrom` is a property name of the foreign key on the "source" model (for
+    example, `customerId` on `Order` model). `keyFrom` is always set to the
+    `decorated property name`.
+  - `keyTo` is a property name on the "target" model, typically the primary key
+    of the "target" model. `keyTo` attribute defaults to the value `id`.
+  - `name` is the target model name. name attribute defaults to
+    `{decorated-property-name}` after stripping the trailing `Id` suffix
+- `property definition`, is an optional field - creates a property decorator
```

Comment:
for consistency: `property definition`, is an optional field -> `property definition` (optional)

Reply:
agree